qsirecon/interfaces/recon_scalars.py

```diff
@@ -270,6 +270,11 @@ class _OrganizeScalarDataOutputSpec(TraitedSpec):
         traits.Str(),
         Undefined,
     )
+    desc = traits.Either(
+        traits.Str(),
+        Undefined,
```

Comment:
this is interesting - I thought that all traits objects are implicitly either `Undefined` or a their actual trait class.

Reply:
I thought they were too, but this interface does return explicit Undefined values instead of just not returning them, so maybe that's why the Either is necessary.